### ISSUE TITLE
Build error can't import iniparser in config.c

### ISSUE THREAD

**Issue by TechAdvancedCyborg** [nikp123/xava#10]
I have used xava on Windows in the past but I recently switched to Ubuntu, I have tried building the project with no success it fails with this error when I run `make -j$(nproc)`:
```
/home/ontake/Downloads/xava/src/config.c:4:11: fatal error: iniparser.h: No such file or directory
    4 |  #include <iniparser.h>
      |           ^~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/xava.dir/build.make:180: CMakeFiles/xava.dir/src/config.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:76: CMakeFiles/xava.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

```
If you could help me that would be great!

**Comment by nikp123** [comment]
First of all, thanks for reporting the issue.

It seems to me that CMake didn't pick up on iniparser properly and one of two things happened:
1. That it's header files are in a subdirectory that I didn't bother to check, as I assume everything is in the root directory of include files (that is ``/usr/include`` or ``/usr/local/include``), you can easily check if this is the case by running: ``find /usr/{include,local/include} | grep iniparser\\\.h`` and you should get either ``/usr/include/iniparser.h`` or ``/usr/local/include/iniparser.h`` as output. If it's not inform me.
2. Or that my iniparser check in the build files has a false positive.
(3. something completely unrelated, but am unware at the moment)

tl;dr: run ```find /usr/{include,local/include} | grep iniparser\\\.h``` and inform me if you don't get either ``/usr/include/iniparser.h`` or ``/usr/local/include/iniparser.h`` as output.

**Comment by TechAdvancedCyborg** [comment]
I ran the command and got: `/usr/include/iniparser/iniparser.h
`

**Comment by nikp123** [comment]
Yup, seems like iniparser is placed in a subdirectory of it's own, unlike on arch (am using it atm).

**Comment by TechAdvancedCyborg** [comment]
Do I just move it?

**Comment by TechAdvancedCyborg** [comment]
or copy it to the following dirs?

**Comment by TechAdvancedCyborg** [comment]
ran `sudo cp /usr/include/iniparser/* /usr/include/
`

**Comment by TechAdvancedCyborg** [comment]
Fixed the issue

**Comment by nikp123** [comment]
It's not a (permanment) solution, as it requires a manual fix/patch. I'll still address this for other Ubuntu users.

**Comment by TechAdvancedCyborg** [comment]
![image](https://user-images.githubusercontent.com/17462236/81288391-d8ffe600-9064-11ea-8cfb-e0ab8a55847a.png)

**Comment by nikp123** [comment]
Also, just a reminder that moving files in order to satisfy a compiler isn't a good practice, it could end up in the package manager breaking further down the line.

**Comment by TechAdvancedCyborg** [comment]
Yeah, but this is a temp solution that should work for now.

**Comment by TechAdvancedCyborg** [comment]
I copied them so it shouldn't break anything else

**Comment by TechAdvancedCyborg** [comment]
Btw, where do I set the config?

**Comment by nikp123** [comment]
Commit aee056a properly resolves the issue, closing.
